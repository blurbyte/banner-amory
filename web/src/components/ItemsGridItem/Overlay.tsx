import styled from 'styled-components';

const Overlay = styled.div`
  position: absolute;
  top: 0;
  left: 0;
  z-index: 999;
  width: calc(200% + 1rem);
  height: 100%;
`;

export default Overlay;
