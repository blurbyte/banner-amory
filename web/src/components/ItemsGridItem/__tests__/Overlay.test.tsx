import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import Overlay from '../Overlay';

test('renders correctly', () => {
  const component = renderer.create(<Overlay />);
  expect(component.toJSON()).toMatchSnapshot();
});
