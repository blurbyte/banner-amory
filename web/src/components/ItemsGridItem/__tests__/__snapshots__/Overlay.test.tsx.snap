// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  position: absolute;
  top: 0;
  left: 0;
  z-index: 999;
  width: calc(200% + 1rem);
  height: 100%;
}

<div
  className="c0"
/>
`;
